components/Form/Button.tsx

```diff
@@ -1,4 +1,12 @@
-export const Button = ({ loading, children, type, ...props }) => (
+import { FC } from 'react';
+
+type Props = {
```

Comment:
Prefer a more specific naming `ButtonProps` f.e. Easier when searching, making it shorter adds no benefit